.travis.yml

```diff
@@ -1,15 +1,24 @@
 language: clojure
 lein: lein2
-install: (cd cloverage && lein2 deps) && (cd lein-cloverage && lein2 deps)
-script: (cd cloverage && lein2 test) && (cd lein-cloverage && lein2 test)
-sudo: false
-cache:
-  directories:
-    - $HOME/.m2
+sudo: true
+
+before_install: yes | sudo lein upgrade
```

Comment:
The `lein` key in travis.yml also supports precise lein versions. That won't update by itself, but it does mean that we can use the `sudo: false` container-based builds, which are faster and updated more regularly.